PSReadLine/CharMap.cs

```diff
@@ -0,0 +1,705 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.PowerShell
+{
+    /// Character sequence translator for platforms that behavior is not
```

Comment:
Why `///`? These aren't XML comments (and they don't need to be.)

Reply:
I do that a lot by accident, I'll change them.